File: src/main/java/com/haozileung/web/service/permission/IElementService.java
package com.haozileung.web.service.permission;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.permission.Element;

public interface IElementService {

    void save(Element dictionary);

    void update(Element dictionary);

    void delete(Element dictionary);

    PageResult<Element> query(Element dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/permission/IElementService.kt
package com.haozileung.web.service.permission

import com.haozileung.infra.pager.PageResult
import com.haozileung.web.domain.permission.Element

interface IElementService {

    fun save(dictionary: Element)

    fun update(dictionary: Element)

    fun delete(dictionary: Element)

    fun query(dictionary: Element, pageNo: Int, pageSize: Int): PageResult<Element>
}
